From 49e88ab4bf571dbfb70fdf7845b402b5aa1ba1c5 Mon Sep 17 00:00:00 2001
Message-ID: <49e88ab4bf571dbfb70fdf7845b402b5aa1ba1c5.1716242564.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1716242563.git.sharlatanus@gmail.com>
References: <cover.1716242563.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 16 May 2024 10:49:38 +0100
Subject: [PATCH 16/26] gnu: python-roman-datamodels: Update to 0.20.0.

* gnu/packages/astronomy.scm (python-roman-datamodels): Update to
0.20.0. Adjust indentation.

Change-Id: I6bd533d81a0c8b6f2cc183e274190ff7042e965f
---
 gnu/packages/astronomy.scm | 40 ++++++++++++++++++++------------------
 1 file changed, 21 insertions(+), 19 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 16e5310e74..b7c885c8f2 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -5098,13 +5098,13 @@ (define-public python-radiospectra
 (define-public python-roman-datamodels
   (package
     (name "python-roman-datamodels")
-    (version "0.19.1")
-    (source (origin
-              (method url-fetch)
-              (uri (pypi-uri "roman_datamodels" version))
-              (sha256
-               (base32
-                "1fp9rwzm5kzm0hjwwmlx7kym7yriyhxbfg2zr3y625pyvwpnl0a8"))))
+    (version "0.20.0")
+    (source
+     (origin
+       (method url-fetch)
+       (uri (pypi-uri "roman_datamodels" version))
+       (sha256
+        (base32 "1918wnssf478w168mhv009jkirmny8hyfxrkwvl8iish36dcqagh"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -5115,18 +5115,20 @@ (define-public python-roman-datamodels
           (add-after 'unpack 'set-env
             (lambda _
               (setenv "HOME" "/tmp"))))))
-    (propagated-inputs (list python-asdf
-                             python-asdf-astropy
-                             python-astropy
-                             python-gwcs
-                             python-numpy
-                             python-psutil
-                             python-rad))
-    (native-inputs (list python-pytest
-                         python-pytest-doctestplus
-                         python-pytest-env
-                         python-pytest-xdist
-                         python-setuptools-scm))
+    (propagated-inputs
+     (list python-asdf
+           python-asdf-astropy
+           python-astropy
+           python-gwcs
+           python-numpy
+           python-psutil
+           python-rad))
+    (native-inputs
+     (list python-pytest
+           python-pytest-doctestplus
+           python-pytest-env
+           python-pytest-xdist
+           python-setuptools-scm))
     (home-page "https://github.com/spacetelescope/roman_datamodels")
     (synopsis "Roman Datamodels Support")
     (description
-- 
2.41.0

